src/test/scala/com/wavesplatform/http/TransactionsRouteSpec.scala

```diff
@@ -51,16 +52,23 @@ class TransactionsRouteSpec extends RouteSpec("/transactions")
     }
 
     "working properly otherwise" ignore {
```

Comment:
Tried to recover this test by creating a mock class which overrides ```Synchronized.read[T](body: ReadLock => T): T```, but the ```500 Error```still come out.